config/config-mysql_apache_jvm.yaml

```diff
@@ -0,0 +1,83 @@
+# This config hopes to mimick the Cloud Monitoring Agent by collecting metrics from popular third-party services the Agent already supports
```

Comment:
`hopes` is not a very optimistic word 😄

Reply:
Should this config file include everything in Jingbo's config file as well?

Reply:
Yeah, we'll merge them into one!

Reply:
I would definitely change the wording to not say `hopes`

Reply:
Yes! forsure, meant to do it last commit